### ISSUE TITLE
Error at fillChains step

### ISSUE THREAD

**Issue by alejandrogzi** [hillerlab/make_lastz_chains#11]
Hi @kirilenkobm,

Since the response of Michael in a previous issue, I successfully masked 3 dog genomes available (35-43%) and the lastz step was done within the expected time (~ 3 days on a 32-core machine). Sadly, I am stuck at the fillChains step with various error exit status. I started running make_lastz_chains with ROS_Cfam_1.0 (NCBI reference; 43% masked) against hg38 as follows: 

`./make_chains.py "human" "dog" "hg38_masked.fa" "ROS_Cfam_1.0.masked.fa"  --queue_executor 32 --project_dir test`

I re-run the job like 3 times after the first error with `--continue_arg "fillChains"` .

This is my log file: [make_chains.log](https://github.com/hillerlab/make_lastz_chains/files/10451793/make_chains.log)

Any advice?

Regards,
Alejandro

**Comment by MichaelHiller** [comment]
The error is that the RepeatFiller script does not know the --workDir parameter. 
The executable is determined by 
my $RepeatFiller = `which RepeatFiller.py`; chomp($RepeatFiller);

--> Can you pls run on your command line 
which RepeatFiller.py

and then check that this is the latest RepeatFiller version? 
3 Months ago, we updated something related to RepeatFiller. 
Therefore, maybe checkout and install the make_chains package from scratch. 
Maybe you are using an older RepeatFiller version on your system that doesn't know this parameter.

**Comment by alejandrogzi** [comment]
Hi @MichaelHiller,

Thanks for the quick response. I did all the install again two times after the first error status without any success before reading your answer here. Following your instructions I surprised myself with the complete absence of RepeatFiller in my system. Seems that the `~/make_lastz_chains/GenomeAlignmentTools` directory was empty. I cloned this repo: https://github.com/hillerlab/GenomeAlignmentTools and moved the .py file to be detected by the runner. Now the "fillChains" step is running correctly (any error status yet).

Thanks for your help,
Alejandro
